examples/platform/k32w/app/support/BUILD.gn

```diff
@@ -14,15 +14,46 @@
 
 import("//build_overrides/chip.gni")
 import("//build_overrides/k32w_sdk.gni")
+import("//build_overrides/pigweed.gni")
+import("$dir_pw_build/facade.gni")
+import("$dir_pw_build/target_types.gni")
+import("$dir_pw_malloc/backend.gni")
+import("$dir_pw_unit_test/test.gni")
 
 config("support_config") {
   include_dirs = [ "../../.." ]
+
+  # Link options that provides replace dynamic memory operations in standard
+  # library with the pigweed malloc.
+  ldflags = [
+    # memory allocation -- these must be re-entrant and do locking
+    "-Wl,--wrap=malloc",
+    "-Wl,--wrap=free",
+    "-Wl,--wrap=realloc",
+    "-Wl,--wrap=calloc",
+    "-Wl,--wrap=MemoryAlloc",
+
+    # Wrap these in case internal newlib call them (e.g. strdup will)
+    # directly call _malloc_r)
+    "-Wl,--wrap=_malloc_r",
+    "-Wl,--wrap=_realloc_r",
+    "-Wl,--wrap=_free_r",
+    "-Wl,--wrap=_calloc_r",
+  ]
+}
+
+pw_facade("pw_malloc") {
```

Comment:
This is not used anywhere and can simply be removed.

Reply:
> This is not used anywhere and can simply be removed.

It's also not the right thing to do even if it were needed... the right thing would be to depend on the common facade and then set pw_malloc_BACKEND. But I think it serves no purpose here.

Reply:
Yes, you're right, I removed pw_facade